selftests/functional/test_basic.py

```diff
@@ -190,6 +190,18 @@ def test_runner_failfast(self):
         self.assertEqual(result.exit_status, expected_rc,
                          "Avocado did not return rc %d:\n%s" % (expected_rc, result))
 
+    def test_runner_force_execution(self):
+        os.chdir(basedir)
+        cmd_line = ('%s run --sysinfo=off --job-results-dir %s '
+                    'passtest.py badtest.py --force-execution on'
```

Comment:
I'd also suggest adding test where all tests are bad to see we don't change the behavior (should not be executed, right?)